api/capi/src/nnstreamer-capi-single.c

```diff
@@ -611,9 +611,11 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     }
   }
 
-  /* set framework, model files and custom option */
+  /* set accelerator, framework, model files and custom option */
   fw_name = ml_get_nnfw_subplugin_name (nnfw);
-  g_object_set (filter_obj, "framework", fw_name, "model", info->models, NULL);
+  hw_name = ml_nnfw_to_str_prop (hw);
+  g_object_set (filter_obj, "accelerator", hw_name, "framework", fw_name, "model", info->models, NULL);
```

Comment:
Q. if `hw_name` is NULL (in case of failure of `ml_nnfw_to_str_prop`), what happens? Are the next arguments following `hw_name` omitted?